testing/meek/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: kpcyrd <git@rxv.cc>
+# Maintainer: kpcyrd <git@rxv.cc>
+pkgname=meek
+pkgver=0.30
+pkgrel=0
+pkgdesc="A pluggable transport proxy written in Go"
+url="https://trac.torproject.org/projects/tor/wiki/doc/meek"
+arch="all"
+license="CC0 1.0 Universal"
+depends="ca-certificates"
+makedepends="go"
+subpackages="$pkgname-doc"
+# no test suite available
+options="!check"
+source="$pkgname-$pkgver.tar.gz::https://gitweb.torproject.org/pluggable-transports/$pkgname.git/snapshot/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname"
+
+prepare() {
+	mkdir -p ${builddir%/*}
+	mv "$srcdir/$pkgname-$pkgver" "$builddir/"
+	cd "$builddir"
+
+	export GOPATH="$srcdir"
+	(cd meek-client && go get -v -d)
+	(cd meek-server && go get -v -d)
```

Comment:
We dont want to run go get in aports as we it doesn't version the deps and fails to create reproducible builds. Instead you should use something like glide to generate a versioned dependency list (lock/yaml file) which is included in the aport.

Reply:
Do you have an example in aports? I'm not meek upstream, glide works by vendoring all dependencies into the repo.